### ISSUE TITLE
getsn returns "Unknown" instead of valid serial number

### ISSUE THREAD

**Issue by lwellerastro** [DOI-USGS/ISIS3#3210]
**ISIS version(s) affected**: 3.7.0-RC, conda version

**Description**  
getsn doesn't return a valid serial number.

**How to reproduce**  
Run getsn on an valid isis cube and it will return "Unknown" as the value.

Under conda version of the release:

> getsn from=I53121006RDR.cub 
> Unknown

Via setisis version of the release:

> getsn from=I53121006RDR.cub 
> Odyssey/THEMIS_IR/1070660659.204

There are a couple of cubes you can try this on under /work/users/lweller/Isis3Tests/GetSN/.

**Additional context**  
From conda environment:

> printenv |egrep ISIS
> ISISROOT=/usgs/cpkgs/anaconda3_linux/envs/isis3.7.0-RC
> ISIS3DATA=/usgs/cpkgs/anaconda3_linux/envs/isis3.7.0-RC/data
> ISIS3TESTDATA=/usgs/cpkgs/anaconda3_linux/envs/isis3.7.0-RC/testData

from setisis environment:

> printenv |egrep ISIS
> ISISROOT=/usgs/pkgs/isis3.7.0-RC/install
> ISIS3DATA=/usgs/cpkgs/isis3/data
> ISIS3TESTDATA=/usgs/cpkgs/isis3/testData

Let me know what other information might be hopeful and if this should have maybe gone under astrodiscuss instead.

**Comment by lwellerastro** [comment]
Ok, this probably needs to be renamed/made into a new issue.

There is nothing under the conda ISIS3DATA area.  It's an empty directory.  Should have probably checked that first.  ISIS3TESTDATA is also empty.

**Comment by jessemapel** [comment]
The activate script probably just needs to point to the appropriate data area.

New release process stuff is happening now, so bear with us for a bit.

**Comment by jessemapel** [comment]
Fixed. I removed the old activate scripts and re-ran isis3varinit.py with the correct data and test data areas.

```
(isis3.7.0-RC) isis3mgr@prog28 # getsn from=/work/users/lweller/Isis3Tests/GetSN/I53121006RDR.cub
Odyssey/THEMIS_IR/1070660659.204
```

**Comment by lwellerastro** [comment]
Looks good, however, tab completion no longer works (it did before).

**Comment by jessemapel** [comment]
@lwellerastro I manually fixed that in our 3.6.2 installation, but it hasn't been fixed for full release yet. See #3163. Is it okay if I tag that as something products wants?

**Comment by lwellerastro** [comment]
It did work for 3.7.0 - I mentioned it in an email to Summer and she took care of it. I honestly don't care whether I run setisis or conda activate (though setisis is easier, less words) - I was just trying to play along w/ the conda stuff since you all seem to want users to get used to it.  I'll just use setisis until 3163 gets resolved.

**Comment by jessemapel** [comment]
@lwellerastro I'll touch base with Summer about this then.

**Comment by lwellerastro** [comment]
@jessemapel , Summer stopped by and mentioned she just forgot to run a script when she made the build.  I can't remember if I emailed her about the problem or if I posted it somewhere.  I tried to conda 3.7.0rc and there were no isis environment variable sets and that got the ball rolling.  I might have been chatting with Kristin about it at the time as well - I don't remember.  Too many places to convey information.

At any rate, I think it is totally fine if we are not ready for conda isis internally due to the setup scripts and tcsh needing the right attention.  I hadn't realized we weren't there yet and when the announcement came out that isis3.7.RC was available, I jumped on it (need to get to jigsaw chagnes) and went the conda route.  When things weren't working there I went back to setisis.  Conveyed my confusion somewhere to someone, and here we are.  No big deal, I can hang via setisis.  Really, I was just trying to support the changes dev is trying to implement.
